src/verinfast/config.py

```diff
@@ -167,6 +167,7 @@ class Config(printable):
     reportId: int = 0
     runDependencies: bool = True
     runGit: bool = True
```

Comment:
Don't mix PRs, this should be in a broader PR with tests and extra configs.

Reply:
Removed